lib/magic_cloud/canvas.rb

```diff
@@ -56,7 +56,11 @@ def render
     private
 
     def set_text_options(draw, options)
-      draw.font_family = options[:font_family]
+      if options[:font_source]
+        draw.font = options[:font_source]
```

Comment:
Don't you think we should name that exactly like option in RMagick is named (e.g. "font")? As far as I understand, it is either family+weight+style, or path to font file (which contains only one weight and style, so setting them in this case is pointless).

Reply:
Thanks, I just updated option name.